orbit/pkg/user/user_notdarwin.go

```diff
@@ -0,0 +1,9 @@
+//go:build !darwin
+// +build !darwin
+
+package user
+
+// CheckUserGuiLoginStatus returns whether or not a user is logged into the machine via the GUI.
```

Comment:
```suggestion
// IsUserLoggedInViaGui returns whether or not a user is logged into the machine via the GUI.
```

Also, I guess we could mention here too that it's only relevant on macOS, and we assume yes for other platforms.

Reply:
Good catch! Yes, will note that in the comment.